ui/app/index.html

```diff
@@ -46,8 +46,11 @@
             <li ng-class="{ active: $state.includes('home.streams') }">
                <a ui-sref="home.streams.tabs.definitions">Streams</a>
             </li>
+            <li ng-class="{ active: $state.includes('home.tasks') }">
+              <a ui-sref="home.tasks.tabs.definitions">Tasks</a>
+            </li>
             <li ng-class="{ active: $state.includes('home.jobs') }">
```

Comment:
@ilayaperumalg @mminella  @cppwfs  Nope - The Jobs tab will contain Job-specific info. In this case Job Executions. Eventually a Job Execution should link back to the corresponding Task Execution. 

A Task-Execution **may** or **may-not** have a corresponding Job Execution but a Job Execution will always have a Task Execution. 

Long-term, we may even re-add more features back to the Jobs tab. There is even the UX team working on this right now to provide more guidance.